.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.2
-before_install: gem install bundler -v 1.16.1
+  - 2.4.5
```

Comment:
no 2.3 or 2.2 ? I think that if this project is trying to promote better gem lib freshness.. I would suspect those projects might be not so fresh with their ruby versions as well.. just a thought..

Reply:
Attempted in a9487b1. Fixed in 6c27f57.